generatedocs/data/config/scheduler/description.md

```diff
@@ -1,3 +1,17 @@
 ## Scheduler section
 
 This section of the JSON file contains scheduler settings for the users in the load scenario.
+
+### Using "reconnectsettings"
```

Comment:
Suggestion:
"### Using `reconnectsettings`"

In addition, it would be great if this section could be placed after the list of scheduler settings.

Reply:
so in the "examples" section then?

Reply:
As it is now, the "Using reconnectsettings" text flows straight into the Schdeuler settings. There should be some way of dividing the two.

![image](https://user-images.githubusercontent.com/22763982/86894889-fbb18800-c103-11ea-970e-0ba321b4d8b8.png)

Reply:
Ah I see, moved it to after the list, into the "examples" section, so might look better

Reply:
Great! Thanks!